apps/studio/pages/api/ai/sql/check-api-key.ts

```diff
@@ -0,0 +1,9 @@
+import { NextApiResponse } from 'next'
+
+export default function handler(res: NextApiResponse) {
+  if (process.env.OPENAI_API_KEY) {
+    res.status(200).send('OK')
+  } else {
+    res.status(401).send('API key not found')
+  }
+}
```

Comment:
this doesn't seem to work (running into 500) and is also not consistent with the way we write our next js endpoints